workspaces/lluis/Image.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="typescript">
+  export let src
+  export let alt
```

Comment:
This could be an issue. Afaik there's no error when it's not provided. It should be a huge red flag thought, becase img tags need to have an alt value. You don't need to solve this issue in this MR, but once this MR is merged can you please create a new issue that asks for creating an eslint rule, which would flag usages of `lluis/Image` that don't have an alt tag provided?

Reply:
Okay got it 👌